### DISCUSSION TITLE
Facing Error on using input CSV files converted to NC files before processing.

### DISCUSSION THREAD

**Discussion by utkarsh-1912** [monocongo/climate_indices#517] [Q&A] [answered]
I converted my CSV files to NC files using xarray but facing below mentioned issue on using as input.
![error_drop_sel](https://github.com/monocongo/climate_indices/assets/79002186/7bdcf50d-ea23-4b8e-a2e4-f6a53769e967)
Please Provide solution.

My Python Code:
` import os
import pandas as pd
import xarray as xr
from netCDF4 import Dataset

input_dir = 'I:/Utkarsh/CSV_Files'
output_dir = 'I:/Utkarsh/NC_Files'

for folder_name in os.listdir(input_dir):
    input_folder_dir = os.path.join(input_dir,folder_name)
    output_folder_dir = os.path.join(output_dir,folder_name)
    os.makedirs(output_folder_dir, exist_ok=True)
    # Iterate through each file in the input directory
    for file_name in os.listdir(input_folder_dir):
        if file_name.endswith('.xlsx') or file_name.endswith('.xls'):
            file_path = os.path.join(input_folder_dir, file_name)
    
            try:
                # Read the Excel file into a DataFrame
                df = pd.read_excel(file_path, index_col='time')
                
                # Convert the DataFrame to an xarray Dataset
                dataset = xr.Dataset.from_dataframe(df)
                
                # Save the Dataset as a NetCDF file
                output_file = os.path.join(output_folder_dir, f'{file_name[:-5]}.nc')
                dataset.to_netcdf(output_file)
            except Exception as e:
                print(f"Error occurred while converting {file_name}: {e}")

print("Conversion completed.") `

**Comment by monocongo** [comment]
My suggestion is to also post your NetCDF file(s) someplace so others can access them, maybe someone will be able to see what the issue is if they can reproduce the issue using your input data.

**Reply by utkarsh-1912** [reply, marked as answer]
Hi,
I am sorry, but the problem is resolved by replacing ".drop_sel" with ".drop_vars" in the "__ main __.py" file as there was an error in the program file, not in the conversion code.

I request you to please update the stable version code of Python package as on pip installation still downloading non-updated version of code.

Btw Thanks for the reply.
